canonical-web-and-design/maas-ui
# Align state models with api

## Done
Align top level state models with API. This allows us to build the rpc method (e.g. `user.list`) from `model` and `method` in action creators.

## QA
Load views, save data, generally use the app. Ensure actions are dispatched appropriately.
